Content.Server/Spreader/KudzuSystem.cs

```diff
@@ -71,6 +74,10 @@ private void OnKudzuSpread(EntityUid uid, KudzuComponent component, ref SpreadNe
         foreach (var neighbor in args.NeighborFreeTiles)
         {
             var neighborUid = Spawn(prototype, neighbor.Grid.GridTileToLocal(neighbor.Tile));
+
+            if (TryComp<KudzuComponent>(neighborUid, out var neighComp))
+                neighComp.SpreadRadius = component.SpreadRadius - 1;
```

Comment:
this code is kinda strange since it doesn't actually track a true center or a distance from that. It's entirely possible to manually decrease the spread radius by just destroying whichever kudzu entity has the highest value.

Reply:
Then what would be the best option?